patches/@stencil+core+4.20.0.patch

```diff
@@ -1,7 +1,18 @@
 diff --git a/node_modules/@stencil/core/compiler/stencil.js b/node_modules/@stencil/core/compiler/stencil.js
-index 05c98c9..652dffe 100644
+index 05c98c9..1ae35e5 100644
 --- a/node_modules/@stencil/core/compiler/stencil.js
 +++ b/node_modules/@stencil/core/compiler/stencil.js
+@@ -234904,8 +234904,8 @@ var getTsOptionsToExtend = (config) => {
+     // if the `DIST_TYPES` output target is present then we'd like to emit
+     // declaration files
+     declaration: config.outputTargets.some(isOutputTargetDistTypes),
+-    module: import_typescript.default.ModuleKind.ESNext,
+-    moduleResolution: import_typescript.default.ModuleResolutionKind.NodeJs,
++    module: import_typescript.default.ModuleKind.ES2022,
++    moduleResolution: import_typescript.default.ModuleResolutionKind.Bundler,
```

Comment:
The new "exports" field in package.json defining the different entry points need the moduleResolution in tsconfig.ts to be either Bundler/Node16/NodeNext . From now on, every user of Headless will have to use one of these three for their typescript to understand where to look for subpackages. [More info on the "exports" field  ](https://nodejs.org/api/packages.html#package-entry-points).

There is a problem with Stencil however. Inside of their compiler, they use an old version of moduleResolution that cant find the subpackages with the "exports" field. This is a [known bug](https://github.com/ionic-team/stencil/issues/4229) and will be fixed in v5.   

This patch above fixes this by changing the default value used by the stencil compiler.